From fc5703ced8c2a66e99f717608c14596482f026a8 Mon Sep 17 00:00:00 2001
From: Ciprian Costea <ciprianmarian.costea@nxp.com>
Date: Thu, 2 Feb 2023 14:33:48 +0200
Subject: [PATCH 24/25] s32cc: tmu: Update 'clks' dts entry

Change the 'tmu' 'clks' directive to a more common usage.

Issue: ALB-9727
Upstream-Status: Pending 

Signed-off-by: Ciprian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32cc.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/s32cc.dtsi b/arch/arm64/boot/dts/freescale/s32cc.dtsi
index dd2288c9693d..afca4a352c69 100644
--- a/arch/arm64/boot/dts/freescale/s32cc.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32cc.dtsi
@@ -277,8 +277,8 @@ tmu: tmu@400a8000 {
 			compatible = "nxp,s32cc-tmu";
 			reg = <0x0 0x400a8000 0x0 0x3000>,
 				<0x0 0x400a4200 0x0 0x1fc>;
-			clocks = <&clks S32CC_SCMI_CLK_TMU_MODULE
-				&clks S32CC_SCMI_CLK_TMU_REG>;
+			clocks = <&clks S32CC_SCMI_CLK_TMU_MODULE>,
+				<&clks S32CC_SCMI_CLK_TMU_REG>;
 			clock-names = "tmu_module", "tmu_reg";
 			status = "okay";
 		};
-- 
2.25.1

